README.md

```diff
@@ -231,10 +231,10 @@ Please make sure you read and observe our [Code of Conduct].
 ## Pull Request process
 
 1. Fork it
-1. Create your feature branch (`git checkout -b feature/fooBar`)
-1. Commit your changes (`git commit -am 'Add some fooBar'`)
-1. Push to the branch (`git push origin feature/fooBar`)
-1. Create a new Pull Request
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Commit your changes (`git commit -am 'Add some fooBar'`)
+4. Push to the branch (`git push origin feature/fooBar`)
+5. Create a new Pull Request
```

Comment:
This seems worse. Did github not render this correctly?

Reply:
I'm seeing this
![image](https://user-images.githubusercontent.com/16121413/53103847-f1352500-34f3-11e9-85e1-2bc9c079209a.png)

where the numbering is rendered when clicking the 'view file' button

Reply:
I think Ashley's talking about the repeated 1s in markdown to make a numbered list.

Reply:
Yeah, sorry, should have been more clear. Markdown will automatically increment a list of `1`s to be however long the list is. I think it is better, because  if you need to add something to the list, you don't have to redo all the numbers, you just add another 1.

Reply:
Ahh, yea that's a nice feature.

I manually changed that markdown from  "1. 1. 1." to "1. 2. 3." (github didn't auto create that)

I can go either way, a list of "1. 1. 1."  or "1. 2. 3."
The advantage that the "1. 2. 3." style has is that it's easily seen how many items are present